cmd/create/idp/htpasswd.go

```diff
@@ -106,21 +107,33 @@ func createHTPasswdIDP(cmd *cobra.Command,
 
 func buildUserList(cmd *cobra.Command, r *rosa.Runtime) *cmv1.HTPasswdUserListBuilder {
 
+	argsCounter := 0
 	// comma separated list of users -  user1:password,user2:password,user3:password
 	users := args.htpasswdUsers
-
+	if len(users) != 0 {
+		argsCounter++
+	}
 	//continue support for single username/password for  backcompatibility
 	// so as not to break any existing automation
 	username := args.htpasswdUsername
 	password := args.htpasswdPassword
+	if username != "" {
+		argsCounter++
+	}
+	usersfile := args.htpasswdFile
+	if usersfile != "" {
+		argsCounter++
+	}
 
-	if username != "" && len(users) > 0 {
-		r.Reporter.Errorf("Only one of  'users' or 'username/password' may be specified. " +
-			"Choose the option 'users' to add one or more users to the IDP.")
+	if argsCounter > 1 {
```

Comment:
I feel like we should move this to a function, like hasMultipleUserArgs or something. Just so it is clearer

Reply:
@gdbranco : ok, I will refractor it.